src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -414,13 +414,27 @@ extern "C" int32_t CryptoNative_GetSslConnectionInfo(SSL* ssl,
     return 0;
 }
 
+// Controls if ERR_clear_error is going to be called before SSL_write/SSL_read
+static bool g_forceOpenSslErrorQueueCleanupBeforeWriteRead = false;
+
+extern "C" void CryptoNative_ForceOpenSslErrorQueueCleanupBeforeWriteRead()
```

Comment:
All of our `extern` functions should also be defined in the corresponding header.

Reply:
No problem, I will follow it for consistency, but I'm a bit curious about the purpose of it. Do you know why?